data/en/listappend.json

```
@@ -22,13 +22,13 @@
 		{
 				"title":"Simple listAppend Example",
 				"description":"Add 'foo' to the end of this list",
-				"code" : "listAppend(\"bar,bar2\", \"foo\")",
+				"code" : "newList = listAppend(\"bar,bar2\", \"foo\")",
```

Maybe this is what you attempted to emphasize:

```suggestion
				"code" : "oldList = \"bar,bar2\";\nnewList = listAppend(oldList, \"foo\");\nwriteOutput(oldList & \"-->\" & newList);",
```

---

I think that would emphasize it, but if you look at the doc for listPrepend you will see that the examples are similar to my edit here. I was merely attempting to make it easier for others to see that unlike arrayAppend and structAppend, listAppend does not modify the list specified but instead returns a copy with the new value appended.

---

Your example didn't have a `writeOutput()` so it caused the tests to fail like Pete mentioned below.